tests/runner.py

```diff
@@ -1354,30 +1354,40 @@ class BrowserCore(RunnerCore):
   def __init__(self, *args, **kwargs):
     super(BrowserCore, self).__init__(*args, **kwargs)
 
+  def get_browser_open_func():
+    if not EMTEST_BROWSER:
+      print("Using default system browser")
+      return webbrowser.open_new
+    browser_args = shlex.split(EMTEST_BROWSER)
+    # If the given browser is just a type, use the built-in functionality
+    # of the webbrowser module to open that type of browser
+    if len(browser_args) == 1 and '/' not in browser_args[0]:
+      print("Using Emscripten browser: " + browser_args[0])
```

Comment:
the prints could be `logger.debug`

Reply:
Done

Reply:
But we were using `print` before.  Why change it.   It seems to me that the browser being used is important information and this is only printed once on class setup right?

Reply:
Sam makes a good point. `print` is also used almost everywhere else in this file instead of `logger.debug`. Should all of those be changed too?

Reply:
I've gone ahead and reverted to plain print statements here and opened a separate pull request (#10699) for updating all of the print statements in the test runner.